examples/objective-c/auth_sample/MakeRPCViewController.m

```diff
@@ -72,8 +83,7 @@ - (void)viewWillAppear:(BOOL)animated {
       }];
 
   // Set the access token to be used.
-  NSString *accessToken = GIDSignIn.sharedInstance.currentUser.authentication.accessToken;
-  call.requestHeaders[@"Authorization"] = [@"Bearer " stringByAppendingString:accessToken];
+  call.oauthToken = [[OAuthAuthenticator alloc] init];
```

Comment:
It'd be nice to have a gRPC+GID category in an optional subspec, so that people using this don't have to write the `OAuthAuthenticator` class above; they'll just call something like `call.useGIDToken` or whatever; just one line of code.